extra/utils/permissions.js

```diff
@@ -0,0 +1,34 @@
+let permissions={
+    'getUsers': {
+    all: ['head-trainer'],
+    read : ['trainee', 'trainer'],
+    write : ['trainer'],
+    delete: [],
+    },
+    'myUsers':{
+        all: ['hod'],
+        read : ['student', 'teacher'],
+        write : ['teacher'],
+        delete: ['admin'],
+    }
+    };
+console.log(hasPermission("getUsers","trainee","read"));
```

Comment:
What's this log for?

Reply:
This is to show the returning value of function hasPermission. Should I remove this?